pkg/controller/utils/credentials.go

```diff
@@ -60,11 +53,9 @@ func ValidateCredentialsForClusterDeployment(kubeClient client.Client, cd *hivev
 
 		}
 
-		return validateVSphereCredentials(cd.Spec.Platform.VSphere.VCenter,
-			string(secret.Data[constants.UsernameSecretKey]),
-			string(secret.Data[constants.PasswordSecretKey]),
-			rootCAFiles,
-			logger)
+		// TODO determine whether any vsphere credential checks are necessary
+		// or if we can remove this function entirely
+		return true, nil
```

Comment:
I don't think we want to merge this piece until we understand the extent of the impact.